commands/utils/utils.go

```diff
@@ -168,3 +169,26 @@ func GetCompatibleOutputWriter(provider vcsutils.VcsProvider) OutputWriter {
 	}
 	return &StandardOutput{}
 }
+
+// RemoveDowngradedVersions removes fix suggestions which are downgraded versions of current
+// trimming brackets as some xray api returns with or without brackets it breaks the compare function
```

Comment:
The above sentence is too long, and therefore hard to understand. Let's split it into shorter digestable sentences.